### DISCUSSION TITLE
Not able to change  label name  in some of the screen Eg: BGMS to FRAC

### DISCUSSION THREAD

**Discussion by sohailamjad12** [Sunbird-Ed/Community#420] [General Discussion]
Hi, Team
In the Question Creation Screen when we click on  **Add to library**  CTA  after redirected to the another screen we are able to see framework as 'Board', 'Medium', 'Grade',Subject". These all value are hardcoded   and comes from package "**@project-sunbird/sunbird-collection-editor-v9**". I need to change it to 
Board -> Function,
Medium -> Role
Grade -> Activity
Subject -> Competency..
Is there any way to changing the label from package?
Iam attaching one attachment for better understanding ..Thanks
@vinukumar-vs @rajeevsathish 
![image-20230220-182006](https://user-images.githubusercontent.com/117971377/221475928-262dcb7f-3316-4674-a78f-5b2e6143ec73.png)

**Comment by chitranshu-keshav** [comment]
@rajeevsathish @vinukumar-vs Can you please get back on the above issue

**Comment by vinukumar-vs** [comment]
@vaivk369 Can you check & confirm these properties? 

@rajeevsathish

**Comment by vaivk369** [comment]
@sohailamjad12 As of now these question properties are hardcoded in the collection editor package. If you want to change the framework properties you would need to modify the code.

**Comment by vaivk369** [comment]
I have created a ticket for this, https://project-sunbird.atlassian.net/browse/KN-844
@pallakartheekreddy @Krishnaj20 Please check.
cc: @vinukumar-vs

**Reply by Krishnaj20** [reply]
@vaivk369 - have you intitiated a discussion thread in Knowlg forums? If not, pls initiate the thread.

@mmanimegalai

**Reply by vaivk369** [reply]
@Krishnaj20 https://github.com/Sunbird-Knowlg/Community/discussions/105 here is the discussion for the same.
